packages/account/src/Components/address-details/address-details.tsx

```diff
@@ -115,7 +106,7 @@ const AddressDetails = ({
         promise.then(() => {
             setHasFetchedStatesList(true);
             if (props.value.address_state) {
-                setAddressStateToDisplay(getLocation(states_list, props.value.address_state, 'text'));
+                setAddressStateToDisplay(getLocation(states_list, props.value.address_state, 'text') ?? '');
```

Comment:
`getLocation` returns an empty string as a default value, do we really need to add this condition here?

Reply:
it also can return undefined. will modify getLocation to avoid it